### DISCUSSION TITLE
minor error in --help

### DISCUSSION THREAD

**Discussion by b1tr4t** [qarmin/czkawka#1035] [General]
Hi, 

help says the `--directories` switch takes a list:

```
$ czkawka dup --help
Finds duplicate files

Usage: czkawka dup [OPTIONS] --directories <DIRECTORIES>

  -d, --directories <DIRECTORIES>
          List of directorie(s) which will be searched(absolute path)

```

But the program demands the switch before each directory:

`$ czkawka -d dir1 -d dir2`

Other subcommands may be similar (I haven't checked).

FYI:
```
$ lsb_release -a  2>/dev/null
Distributor ID:	Debian
Description:	Debian GNU/Linux 12 (bookworm)
Release:	12
Codename:	bookworm
```
```
$ git remote show origin
* remote origin
  Fetch URL: https://github.com/qarmin/czkawka.git
  Push  URL: https://github.com/qarmin/czkawka.git
  HEAD branch: master
  Remote branches:
    3.3.2     tracked
    master    tracked
    webp_heif tracked
```

A very minor complaint!  Awesome package thanks.  :-)

Cheers,
bitrat

**Comment by rugk** [comment]
@b1tr4t The project is open-source, so you may adjust the code to fix minor issues like that directly.

I guess here is where you want to start:
https://github.com/qarmin/czkawka/blob/8df5e991a6f05e49069a447d7d300de433ed8f2f/czkawka_cli/src/commands.rs#L619-L627
